src/components/structures/RoomSubList.js

```diff
@@ -347,8 +347,14 @@ var RoomSubList = React.createClass({
         var label = this.props.collapsed ? null : this.props.label;
 
         let content;
-        if (this.state.sortedList.length === 0 && !this.props.searchFilter && this.props.extraTiles.length === 0) {
-            content = this.props.emptyContent;
+        if (this.state.sortedList.length === 0) {
+            // check if emptyContent is defined, if falsey then don't show an empty sublist even without searchFilter
```

Comment:
To me, this comment makes things more confusing. I would at least write it in the order of conditions, i.e. "If no search filter is applied and ..." or "Only show placeholder if ..."

Reply:
Done thanks